departures.go

```diff
@@ -68,6 +70,22 @@ func main() {
 		return
 	}
 
+	// search of the station and provide user option to choose
+	if id == "" && stationName != "" {
+		s, err := promptForStation(stationName)
+		if err != nil {
+			fmt.Println(err)
+		} else {
+			id = s.ID
+		}
+	}
+
+	// set default id if empty
+	if id == "" {
+		fmt.Println("station id is empty. Defaulting to: 900000100003")
```

Comment:
Could you please capitalize "ID", makes the messages more consistent.

Reply:
Updated